### DISCUSSION TITLE
Both authorize.net modules use deprecated APIs

### DISCUSSION THREAD

**Discussion by cstdenis** [zencart/zencart#5819] [Ideas]
Both authorize.net modules use depreciated APIs. This should be replaced in the future if authorize.net continues to be one of the processors listed as supported as listed here: https://www.zen-cart.com/content.php?14-Payment-Processing

https://developer.authorize.net/api/upgrade_guide.html

**Comment by scottcwilson** [comment]
Do you have the skills to do this work?  Your help would be much appreciated if so.

**Comment by proseLA** [comment]
auth.net has been talking about these deprecated methods for many years now.  

in my conversations with chris brown, he was of the opinion that auth.net gets too much business using these methods that they will not stop taking transactions from them for the foreseeable future.  that conversation was 3+ years ago.  so far, he is proven correct.

if auth.net does in fact stop taking transactions using these methods, [this module](https://www.zen-cart.com/downloads.php?do=file&id=2272) makes use of the auth.net api.  which is not deprecated.
